src/services/Storage.ts

```diff
@@ -17,3 +24,92 @@ export const createNote = async (
   };
   return newPostFeedItem;
 };
+
+export class Store implements StoreInterface {
+  put(targetPath: string, _content: Content): Promise<URL> {
+    return Promise.resolve(
+      new URL(`${process.env.REACT_APP_UPLOAD_HOST}/${targetPath}`)
+    );
+  }
```

Comment:
just fyi, we might not need to have this soon: https://github.com/LibertyDSNP/sdk-ts/pull/96